### PR TITLE
minting fee

### PR DESCRIPTION
**Pull request by ETeissonniere**

### CODE REVIEW

File: pallets/nfts/src/default_weights.rs [hunk 1]
```diff
@@ -1,39 +1,135 @@
-use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
+/*
+ * This file is part of the Nodle Chain distributed at https://github.com/NodleCode/chain
```
Comment (markopoloparadox) [inline pallets/nfts/src/default_weights.rs:2]: Is this correct?

**Review by markopoloparadox — COMMENTED** [review]
Just found one observation and that's all. lgtm.

File: pallets/nfts/src/default_weights.rs (hunk 1 above)
Comment (ybensacq) [reply pallets/nfts/src/default_weights.rs:2]: No :-)

Comment (ETeissonniere) [reply pallets/nfts/src/default_weights.rs:2]: Oh crap, that comes from the template I used

Comment (ETeissonniere) [reply pallets/nfts/src/default_weights.rs:2]: I had added a weights template for when we generate them

Comment (ETeissonniere) [reply pallets/nfts/src/default_weights.rs]: Should be fixed :)

Comment (markopoloparadox) [reply pallets/nfts/src/default_weights.rs]: Thanx for the quick fix :heart:

**Review by ybensacq — APPROVED** [review]

**Review by markopoloparadox — APPROVED** [review]

**Comment by markopoloparadox** [comment]
It took me a while to find out how to approve PRs xd

**Pull request merged**